test/error_messages.coffee

```diff
@@ -1664,25 +1664,26 @@ test 'CSX error: invalid attributes', ->
 
 test '#5034: CSX error: Adjacent JSX elements must be wrapped in an enclosing tag', ->
   assertErrorFormat '''
-    render = ->
+    render = -> (
       <Row>a</Row>
       <Row>b</Row>
+    )
   ''', '''
     [stdin]:3:4: error: Adjacent JSX elements must be wrapped in an enclosing tag
       <Row>b</Row>
        ^^^^^^^^^^^
   '''
   assertErrorFormat '''
     render = -> (
+      a = "foo"
       <Row>a</Row>
       <Row>b</Row>
     )
   ''', '''
-    [stdin]:3:4: error: Adjacent JSX elements must be wrapped in an enclosing tag
+    [stdin]:4:4: error: Adjacent JSX elements must be wrapped in an enclosing tag
       <Row>b</Row>
        ^^^^^^^^^^^
```

Comment:
@zdenko We should also add a test for the non-error case:

```coffee
->
  <a />
  <b />
```

That would be a good place to put a comment with a link to this PR, and an explanation for why this shouldn’t error.